tensorbay/opendataset/VGGFace2/loader.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+
+"""Dataloaders of VGGFace2 dataset."""
+
+import os
+from collections import defaultdict
+from typing import Any, DefaultDict, Dict, List, Tuple, Union
+
+import numpy as np
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.label import Catalog, Classification, LabeledBox2D, LabeledKeypoints2D
+from tensorbay.opendataset._utility import glob
+from tensorbay.utility import chunked
+
+DATASET_NAME = "VGGFace2"
+_SEGMENTS = {"train": True, "test": False}
+
+
+class_attrs = ["Sample_num", "Flag", "Gender"]
+
+box2d_attrs = [
+    "01-Male",
+    "02-Black_Hair",
+    "03-Brown_Hair",
+    "04-Gray_Hair",
+    "05-Blond_Hair",
+    "06-Long_Hair",
+    "07-Mustache_or_Bread",
+    "08-Wearing_Hat",
+    "09-Eyeglasses",
+    "10-Sunglasses",
+    "11-Mouth_Open",
+]
+
+keypoints = ("L_eye", "R_eye", "Nose", "L_mouth_corner", "R_month_corner")
+
+box2d_files = ("label/loose_bb_test.csv", "label/loose_bb_train.csv")
+ketpoints2d_files = ("label/loose_landmark_test.csv", "label/loose_landmark_train.csv")
+
+
+_NEW = Dict[str, Union[str, int, float, bool, List[Union[str, int, float, bool]]]]
```

Comment:
what is `_NEW` mean?

Reply:
It is a new type of variable type of a function's return value, and it makes more concise for the type declaration.

Reply:
But `_NEW` is not a good variable name because I cannot easily get the meaning of the variable from its name.
Maybe you need to find out a better name for it.